### PR TITLE
Create test util methods to clear interaction states

### PR DESCRIPTION
**Pull request by yiqunw700**

### CODE REVIEW

File: designcompose/src/testFixtures/java/com/android/designcompose/TestUtils.java [hunk 1]
```diff
@@ -39,4 +43,17 @@ public static void triggerLiveUpdate() {
 
         testOnlyTriggerLiveUpdate(actualFigmaToken);
     }
+
+    public static void clearInteractionStates() {
+        InteractionStateManager.INSTANCE.getStates().clear();
```
Comment (rylin8) [inline designcompose/src/testFixtures/java/com/android/designcompose/TestUtils.java:48]: where does `INSTANCE` come from?

Comment (yiqunw700) [reply designcompose/src/testFixtures/java/com/android/designcompose/TestUtils.java:48]: yeah this is java, not kt. Would you like it converted to kt?

Comment (timothyfroehlich) [reply designcompose/src/testFixtures/java/com/android/designcompose/TestUtils.java:48]: When I wrote this there actually wasn't support for Kotlin test fixtures. Looks like it's coming in AGP 8.5: https://b.corp.google.com/issues/259523353?pli=1

Comment (rylin8) [reply designcompose/src/testFixtures/java/com/android/designcompose/TestUtils.java:48]: Oh -- didn't notice. I guess that's fine... I don't know why this file was made in java to begin with -- @timothyfroehlich ?

Comment (timothyfroehlich) [reply designcompose/src/testFixtures/java/com/android/designcompose/TestUtils.java:48]: I replied right before you did :)

**Review by timothyfroehlich — COMMENTED** [review]
This looks awesome. I didn't realize something like this was so simple, I haven't dug into Junit rules that much!

**Review by rylin8 — APPROVED** [review]

**Pull request merged**